api/controller/psmdb_cluster_api.proto

```diff
@@ -96,25 +92,19 @@ message UpdatePSMDBClusterRequest {
       // Requested compute resources.
       ComputeResources compute_resources = 1;
     }
+    // Update cluster size.
+    bool update_cluster_size = 5;
```

Comment:
Do we support updating size to 0? I think we shouldn't, we can just check for non-zero value.

Reply:
Yeah, make sense for now.

Reply:
Or we can allow scaling to 0 to implement cluster pausing without a separate field